libs.autobuild

```diff
@@ -2,7 +2,54 @@
 
 cmake_package 'slam/threed_odometry'
 cmake_package 'gui/pose3d_editor'
+
 ruby_package 'control/ruby_sdformat'
+cmake_package "drivers/glfw" do |pkg|
+    pkg.define "BUILD_SHARED_LIBS", "ON"
+end
+
+#Needed for kinect, the os packages does not (yet) work 06.15
+Autoproj.add_osdeps_overrides 'glfw', :package => 'drivers/glfw'
+remove_from_default 'drivers/glfw'
+
+import_package "drivers/glew"  do |pkg|
+    pkg.depends_on "glfw"
+    pkg.depends_on "libturbojpeg"
+    pkg.post_install do
+        Autoproj.env_set_path "GLEW_DEST", Autobuild.prefix
+        pkg.in_dir(pkg.srcdir) do
+            Autobuild::Subprocess.run(pkg, 'make extensions', Autobuild.tool(:make), 'extensions')
```

Comment:
Use Autobuild.make_subcommand(pkg, 'build', 'extensions') to properly use autoproj's parallel build

 - pkg.in_dir should in general not be used, use pkg.run directly (which sets up the working directory properly)
 - Subprocess.run should not be invoked manually (use pkg.run)
 - the phase should really be 'build' or 'install' to gather the information in the appropriate log files.
 - there is usually a default target, you could do make extensions at the same time than the default target (again, to use the parallel build properly)
 - this will NOT trigger any rebuild. Moreover, the status of #post_install is pretty not-well-defined (for instance as whether it is OK to build there or just to do environment updating). Best would be to redefine the install method:

~~~
def pkg.install
end
~~~

Even better would be to define prepare, build and install and properly set up the things so that the package gets rebuilt on update (it won't in this setup). See Autobuild::CMake or Autobuild::Configurable for more details.